src/lib/utils/meta_tables/meta_log_table.cpp

```diff
@@ -28,14 +28,13 @@ std::shared_ptr<Table> MetaLogTable::_on_generate() const {
     // We need this to format the timestamp in a thread-safe way.
     // https://stackoverflow.com/questions/25618702/
     //   why-is-there-no-c11-threadsafe-alternative-to-stdlocaltime-and-stdgmtime
-    std::ostringstream timestamp;
-    auto time = std::chrono::system_clock::to_time_t(entry.timestamp);
+    auto timestamp_stream = std::ostringstream{};
+    auto timestamp = std::chrono::system_clock::to_time_t(entry.timestamp);
+    auto buffer = tm{};
```

Comment:
What's tm?

Reply:
> Structure holding a calendar date and time broken down into its components.

https://en.cppreference.com/w/c/chrono/tm

Reply:
IFF you're going to touch this PR, a short comment would be nice.

Besides that, Knüller zünden?

Reply:
Can touch this when allowing queries on exported tables / add a switch to export logs on benchmarks. Ready for Zündung if this is fine for you.

Reply:
Fine with me. Zünd it!